docs/design/analytics.md

```diff
@@ -48,7 +48,6 @@ Note: There's a limit of 20 custom dimensions.
 | 5 | `Flag: --style` | `string` |
 | 6 | `--collection` | `string` |
 | 7 | `--buildEventLog` | `boolean` |
-| 8 | `Flag: --enableIvy` | `boolean` |
```

Comment:
Need to inform @StephenFluin about this, if we are going to remove it.

Reply:
But the flag itself no longer exists on `ng generate application`, hence there is no data to collect. 

`enableIvy` exists on `ng new` and `ng generate workspace`

Reply:
Yes, that's true. This file doesn't mention anything about what commands capture it though.

Reply:
yeah, I will add this to `ng-new` to that this matrix is captured there.

Reply:
https://github.com/angular/angular-cli/pull/15186/files#diff-e1a844356231f48df29c54e39fcb9951R25